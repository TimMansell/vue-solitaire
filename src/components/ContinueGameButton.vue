<template>
  <Button route="home" data-test="continue-game-btn"> Continue </Button>
</template>

<script>
import Button from './Button.vue';

export default {
  name: 'ContinueGameButton',
  components: {
    Button,
  },
};
</script>
