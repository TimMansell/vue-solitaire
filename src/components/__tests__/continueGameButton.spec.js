import { describe, it, expect } from 'vitest';
import { shallowMount } from '@vue/test-utils';
import ContinueGameButton from '@/components/ContinueGameButton.vue';

describe('ContinueGameButton.vue', () => {
  it('renders the component without crashing', () => {
    const wrapper = shallowMount(ContinueGameButton);

    expect(wrapper.isVisible()).toBe(true);
  });
});
