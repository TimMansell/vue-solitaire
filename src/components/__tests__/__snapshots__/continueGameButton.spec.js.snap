// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContinueGameButton.vue matches snapshot 1`] = `<button-stub type="default" size="md" topage="home" data-test="continue-game-btn"> Continue </button-stub>`;
